src/lib/es2020.intl.d.ts

```diff
@@ -71,16 +73,9 @@ declare namespace Intl {
     type RelativeTimeFormatStyle = "long" | "short" | "narrow";
 
     /**
-     * [BCP 47 language tag](http://tools.ietf.org/html/rfc5646) definition.
+     * The locale or locales to use
      *
-     * [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#locales_argument).
-     */
-    type BCP47LanguageTag = string;

```

Comment:
why remove BCP47LanguageTag?

Reply:
From the PR topic comment (I should probably have posted inline too to make it more findable)

> I've also removed the type `BCP47LanguageTag` in favor of `UnicodeBCP47LocaleIdentifier` — not sure why there were 2 distinct types for this in the first place, especially given that both are just aliases of `string`. In any case, the correct term is "locale identifier", not "language tag" — BCP 47 locale identifiers are a subset of BCP 47 language tags, and language tags that aren't locale identifiers aren't supported by `Intl`. `BCP47LanguageTag` could alternatively be deprecated rather than removed, to mitigate the unlikely case of userland code using the `Intl.BCP47LanguageTag` type... not sure if that's a necessary consideration, though.

Reply:
Thanks, I missed that.
Let's try taking it out and see if it causes problems. If it does, we can reinstate it, but deprecated.